### PR TITLE
Update multiregion TPCC instructions

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-3353

### CODE REVIEW

**Comment by rmloveland** [comment]
@srosenberg I wasn't able to trigger the dynamic local IP using `cockroach demo` on my Linux system, but I went ahead and added a note that is hopefully general enough to cover that case.  

Please let me know what you think!

**Review by srosenberg — APPROVED** [review]
Looks good, thanks for updating the docs!

File: v22.1/performance-benchmarking-with-tpcc-local-multiregion.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ cockroach demo --global --nodes 9 --no-example-database --insecure
 
 This simulated multi-region deployment will take advantage of CockroachDB's [multi-region SQL statements](multiregion-overview.html) to deliver improved ease of use and performance.
 
+{{site.data.alerts.callout_info}}
+You must use the IP address shown at the SQL prompt to run the following steps.
```
Comment (lnhsingh) [inline v22.1/performance-benchmarking-with-tpcc-local-multiregion.md:41]: The added callout looks good as-is.

One suggestion to clarify further (take it or leave it): add what the `cockroach demo` response looks like and include the sample IP address

**Review by lnhsingh — APPROVED** [review]

File: v22.1/performance-benchmarking-with-tpcc-local-multiregion.md (hunk 1 above)
Comment (rmloveland) [reply v22.1/performance-benchmarking-with-tpcc-local-multiregion.md:41]: Thanks Lauren!  I think I'll pass for now as I would rather users refer to the `cockroach demo` docs which have that info

**Comment by rmloveland** [comment]
Thanks for the reviews @srosenberg and @lnhsingh !

**Pull request merged**